tests/opera/parser/tosca/v_1_3/test_service_template.py

```diff
@@ -1,8 +1,12 @@
 import pytest
+import pathlib
 
 from opera.error import ParseError
 from opera.parser.tosca.v_1_3.service_template import ServiceTemplate
+from opera.parser.tosca.v_1_3.import_definition import ImportDefinition
 from opera.parser.yaml.node import Node
+from opera.parser.tosca.path import Path
+from opera.parser import tosca
```

Comment:
Again, do we need those two imports?

Reply:
No, we don't. Imports were optimized and now it should be ok.